meerk40t/core/svg_io.py

```diff
@@ -683,10 +684,18 @@ def process(self, svg, pathname):
     def check_for_mk_path_attributes(self, node, element):
         for prop in element.values:
             lc = element.values.get(prop)
-            # print (f"Property: {prop} = {lc}")
             if prop.startswith("mk"):
+                # print (f"Property: {prop} = [{type(lc).__name__}] {lc}")
                 if lc is not None:
                     setattr(node, prop, lc)
+                    # This needs to be done as some node types are not based on Parameters
+                    # and hence would not convert the stringified tuple
```

Comment:
Arent the functional parameters inherent to the node types? Why does it need to be done at all?